broker/broker.go

```diff
@@ -0,0 +1,36 @@
+package broker
+
+var (
+	List   ObjectType = "list"
+	Single ObjectType = "single"
+
+	Add    EventType = "ADDED"
+	Update EventType = "MODIFIED"
+	Delete EventType = "DELETED"
+	Error  EventType = "ERROR"
+)
+
+type ObjectType string
+type EventType string
+
+type Message struct {
```

Comment:
A note to be working toward a common message struct - https://docs.google.com/document/d/1_zpEWBcC6ngOLen_E_nc-octiUdBbpmdwwIX6qzo70M/edit#heading=h.3gmuxn6p14m9